server/planning/content_api/types/planning.py

```diff
@@ -0,0 +1,59 @@
+from superdesk.core.resources import fields, Dataclass
+from pydantic import Field
+from superdesk.utc import utcnow
+from datetime import datetime
+from planning.types import PlanningResourceModel, PLANNING_EVENT_LINK_METHOD
+
+
+class AgendaItem(Dataclass):
+    qcode: fields.Keyword
+    name: str
+
+
+class MatchingProduct(Dataclass):
+    code: fields.Keyword
+    name: str
+
+
+class RelatedEvent(Dataclass):
+    _id: fields.Keyword
+    recurrence_id: str | None = None
+    link_type: PLANNING_EVENT_LINK_METHOD | None = None
+
+
+class CoverageAssignedTo(Dataclass):
```

Comment:
This is missing quite a few fields. see [expand_delivery](https://github.com/superdesk/superdesk-planning/blob/async/server/planning/output_formatters/json_planning.py#L162) and [expand_coverage_details](https://github.com/superdesk/superdesk-planning/blob/async/server/planning/output_formatters/json_planning.py#L223) for the extra fields that must be added to the PlanningCoverageItem model

Reply:
done